### ISSUE TITLE
Need help using

### ISSUE THREAD

**Issue by dylanschults** [austinwagner/discord_xbl_status#3]
Is there any way you could maybe make a tutorial on how to use your program? 
I'm kinda new to python and would really like to use this program.

**Comment by Echtze1t** [comment]
Yes, I'd like a step-by-step tutorial as well. I tried to follow the steps in the readme numerous times on multiple different computers, but never got it to work.
